osgar/drivers/kloubak.py

```diff
@@ -266,8 +266,9 @@ def __init__(self, config, bus):
         self.last_pose_encoders = [0, 0,] * self.num_axis  # accumulated tacho readings
         self.encoders = [0, 0,] * self.num_axis  # handling 16bit integer overflow
         self.last_encoders_16bit = None  # raw readings
-        self.last_join_angle = None
-        self.last_join_angle2 = None
+        self.last_join_angle = None  # join angle in radians
```

Comment:
I know that the name is propagation from history, but `joint` would be much better.

Reply:
OK, I missed it